lib/sidekiq/cli.rb

```diff
@@ -269,6 +269,19 @@ def validate!
         logger.info @parser
         die(1)
       end
+
+      errors = []
```

Comment:
Isn't it a nicer user experience to see all errors at once? For me as a user I prefer that instead of fix, fail, fix, fail ..